changelog.d/13346.misc

```diff
@@ -0,0 +1 @@
+Fix long-standing bug where Synapse sometimes asked many different remote hosts for the same event.
```

Comment:
I don't think this is correct. It never caused a user-visible bug, because `get_pdu` is only ever called with a single server.